### PR TITLE
Add base sdk logic supporting query and segment methods

### PR DESCRIPTION
**Pull request by Rodri77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+- `sqlQuery`: SQL query string
+
+Returns a Promise that resolves with the query results.
+
+## Code Snippet for Node.js
+
+Here's a standalone code snippet you can use as a starting point:
+
+```
+const EvefanSDK = require("@evefan/sdk-js").default;
```
Comment (pellicceama) [inline README.md]: I think you can just do import EvefanSDK from '@evefan/sdk-js'. Assuming the file is .cjs and not .js. Lets change it in the docs

Comment (pellicceama) [inline README.md]: Can we call the exported object just 'Evefan' and not EvefanSDK?

**Review by pellicceama — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rodri77) [reply README.md]: the generated file is js and had some issues on the import without the default

Comment (pellicceama) [reply README.md]: that's because you're using it from a js file that uses require and not import. That's ok to change for the example dw :)

Comment (Rodri77) [reply README.md]: Import solved in the last commit

Comment (Rodri77) [reply README.md]: Updated in the last commit.

**Review by pellicceama — APPROVED** [review]
LGTM, I'll follow up with a quick PR later today/tomorrow with some tweaks.

**Pull request merged**